exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/OtlpStdoutExporterBuilderUtil.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.logging.otlp.internal;
+
+import io.opentelemetry.sdk.common.export.MemoryMode;
+
+public class OtlpStdoutExporterBuilderUtil {
+  public OtlpStdoutExporterBuilderUtil() {}
+
+  public static void validate(MemoryMode memoryMode, boolean wrapperJsonObject) {
+    if (memoryMode == MemoryMode.REUSABLE_DATA && !wrapperJsonObject) {
+      throw new IllegalArgumentException(
+          "Reusable data mode is not supported without wrapperJsonObject");
```

Comment:
Let's not extract a helper function for this. We've discussed this idea in the past of whether repeated code should always be extracted to a helper function. For builders, we decided its often better to repeat the simple validation steps so that readers inspecting the code can understand the requirements / logic at a glance without having to dive down another layer of code.